### ISSUE TITLE
[CST] Create Zero Silent Failure logging to support DD dashboards/alerts

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#94405]
## Background Context

- We want to log every time we send a failure email
- We want to know when our attempts to send an email have failed
- We want to have a running log of how many jobs get checked each time the polling solution calls LH
- We want to know when we don't get a polling response back from LH
- We want to know when rows get deleted from the evidence submission table and how many

## Engineering 
- Review files in [this feature branch](https://github.com/department-of-veterans-affairs/vets-api/tree/feature/updated-es-error-handling) and look for commented out logger code, statsd logic and any todos mentioned in the code
-  Use standards in [this doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/platform/practices/zero-silent-failures/logging-silent-failures.md) and conform to best practices

## Out of scope
- Add here

## Open questions
- Add here

## Tasks
- [x] After reviewing docs, sync on what stats to implement
- [x] Define benchmarks for triggering silent failure alerts

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [x] Logging exists for each file related to new silent failure features (polling, exhaustion handler).
- [ ] We have requirements for DataDog dashboards and Slack alerts 
- [ ] We have created two tickets: 1) Silent failure DataDog dashboard and 2) Silent failure Slack error monitoring alerts

**Comment by pmclaren19** [comment]
This is related to [BMT - add custom metrics to track BMT failure items](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94325)

**Comment by pmclaren19** [comment]
Blocked by https://github.com/department-of-veterans-affairs/va.gov-team/issues/100037
